man/systemd.link.xml

```diff
@@ -312,11 +314,10 @@
       <varlistentry>
         <term><varname>Name=</varname></term>
         <listitem>
-          <para>The interface name to use in case all the
-          policies specified in
-          <varname>NamePolicy=</varname> fail, or in case
-          <varname>NamePolicy=</varname> is missing or
-          disabled.</para>
+          <para>The interface name to use. This option has lower priority than
```

Comment:
doesn't really matter, but I'd probably use "takes precedence over" or "is overriden by" or so instead of "has lower priority" for something that doesn't actually know  a numeric (or equivalent) priority concept — in particular as we also have a proper priority concept here in the naming of the .link files, hence this nicely avoids some confusion I think